internal/api/v1beta1connect/role.go

```diff
@@ -197,6 +197,13 @@ func (h *ConnectHandler) CreateOrganizationRole(ctx context.Context, request *co
 		return nil, connect.NewError(connect.CodeInvalidArgument, ErrBadBodyMetaSchemaError)
 	}
 
+	// Fetch organization to get name for audit record
+	org, err := h.orgService.Get(ctx, request.Msg.GetOrgId())
+	if err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+	metaDataMap["org_name"] = org.Title
```

Comment:
Let's hardcode this metadatamap key "org_name" as a const as it is being used at multiple places.